src-cpp/app/boot/app_controller.h

```diff
@@ -1,5 +1,4 @@
-#ifndef APP_CONTROLLER_H
-#define APP_CONTROLLER_H
+#pragma once
```

Comment:
its should be used in whole project and can be selected for IDE (no points to call it "improve", but in projects where we have the same name of most of files `model, view, controller` its necessary)